src/app/integrations/create-page/create-page.component.html

```diff
@@ -13,8 +12,11 @@
       <router-outlet></router-outlet>
       <!-- Wizard Footer -->
       <div class='wizard-pf-footer'>
-        <button type='button' class='btn btn-default btn-cancel wizard-pf-cancel wizard-pf-dismiss'>Cancel</button>
+        <button type='button' class='btn btn-default btn-cancel wizard-pf-cancel wizard-pf-dismiss' (click)="cancel()">Cancel</button>
+        <button type="button" class="btn btn-primary" [disabled]="!canContinue" (click)="continue()" *ngIf="!(getCurrentChild() === 'connection-configure' && position === 1)">Next</button>
+        <!-- TODO hiding showing hack -->        
+        <button type="button" class="btn btn-primary" *ngIf="getCurrentChild() === 'connection-configure' && position === 1" (click)="finish()">Finish</button>
```

Comment:
absolutely, this template is still in the POC phase, I'd  expect to be reworking this, hence the TODO in there...